<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorWidget</class>
 <widget class="QWidget" name="OperatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="volumeSlide">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>20</y>
     <width>25</width>
     <height>161</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QDial" name="coarseDial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="coarseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Coarse</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="fineDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-128</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fineLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="attackDial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="decayDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="sustainDial">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="releaseDial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="attackLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="decayLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Decay</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sustainLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sustain</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="releaseLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Release</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>228</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="absoluteButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="absoluteLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="LFOSpeedDial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LFOSpeedLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LFO Speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="LFOAmountDial">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LFOAmountLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LFO Amount</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="feedbackDial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="trackSoftHighButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>135</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/voltrack-a.png</normaloff>:/images/res/voltrack-a.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="trackSoftLowButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>195</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/voltrack-c.png</normaloff>:/images/res/voltrack-c.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="trackHardHighButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>165</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/voltrack-b.png</normaloff>:/images/res/voltrack-b.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="trackHardLowButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>225</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/res/voltrack-d.png</normaloff>:/images/res/voltrack-d.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="trackingLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="initLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Env. Init</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="initDial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>49</width>
     <height>49</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="notchTarget">
    <double>16.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="coarseLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fineLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="volumeLine">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>195</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Envelope" name="envelopeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Envelope</class>
   <extends>QWidget</extends>
   <header>envelope.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
